### DISCUSSION TITLE
Errors in exception.log file

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1905] [Questions & Answers]
This post aims to centralize the errors that occur in the **exception.log file** in production Magento/OpenMage installations. 

If there are solutions, please do not hesitate to offer them.

**Comment by addison74** [comment]
Here are the errors I've kept in this file over time.

```2020-11-05T01:30:59+00:00 ERR (3): 
Mage_Core_Exception: Invalid webservice adapter specified. in /home/mydomain/public_html/app/Mage.php:595
Stack trace:
#0 /home/mydomain/public_html/app/code/core/Mage/Api/Model/Server.php(127): Mage::throwException('Invalid webserv...')
#1 /home/mydomain/public_html/api.php(84): Mage_Api_Model_Server->initialize(NULL)
#2 {main}

**Reply by luigifab** [reply]
We have same, some robots try api urls, for example (from Sentry):
- xalrk.com/api/version/getVersionByType
- ball.vip/api/openapi/common/getDownloadUrl
- apibx.bongx9.com/api/login

**Reply by luigifab** [reply]
The simple solution: change _api.php_ by _something.php_.
And do not share _something.php_ with google.
Edit: this require some changes in PHP code...
